nix/all-toplevel-derivations.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, pkgSet
+, fn
+, recursionDepth
+, keyFilter
+, ...
+}:
+let
+  go = depth: set':
+    let
+      canSet = builtins.tryEval set';
```

Comment:
Hm, I've got troubles to understand why you've chosen the name `canSet` here. :thinking: 

Wouldn't this be better named the `evaluatablePkgsSet` or `evaluatableSet`? :thinking:

Reply:
Yuh the naming is awful. I will switch to one of your proposed ones 😅